### PR TITLE
H2turbine documentation addition. 

### PR DESCRIPTION
**Pull request by klfrick2**
## Changes proposed in this PR:
-
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md and COPYRIGHT.md file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by andrewlee94 — APPROVED** [review]

**Review by andrewlee94 — APPROVED** [review]

File: docs/models/nuclear/pem_electrolyzer.rst [hunk 1]
```diff
@@ -1,37 +1,42 @@
 PEM Electrolyzer
 ================
 
-The DISPATCHES PEM Electrolyzer Model
+The DISPATCHES PEM Electrolyzer Model represents a simple proton-exchange membrane electrolyzer that operates with a
+fixed efficiency in converting electricity to hydrogen gas. The outlet state is determined via an H2 property package.
+
+This file here is a synposis of the PEM system. The full description with arguments and details on the class are available in
```
Comment (lbianchi-lbl) [inline docs/models/nuclear/pem_electrolyzer.rst:7]: Typo: synposis -> synopsis (just caught my eye during the dev call)

**Comment by ksbeattie** [comment]
@klfrick2 any news on this?

**Comment by ksbeattie** [comment]
@klfrick2 is this still something for the "July (internal) release"?

**Review by ksbeattie — APPROVED** [review]
LGTM

**Review by joshua-cogliati-inl — APPROVED** [review]
I have reviewed the changes and I approve.

**Pull request merged**